setup.sh

```diff
@@ -12,14 +12,19 @@ if [ ! -e ${ADAFRUIT_DIR} ]; then
 	exit 2
 fi
 
-apt-get install -y python python-dev python-pip python-smbus lighttpd
+apt-get install -y python python-dev python-pip python-smbus nginx uwsgi uwsgi-plugin-python
 pip install flask
 
 pushd ${ADAFRUIT_DIR} > /dev/null
 python setup.py install
 popd > /dev/null
 
-cp www/lighttpd.conf /etc/lighttpd/lighttpd.conf
+rm -rf /var/www/rovercode > /dev/null
+mkdir /var/www/rovercode
+ln -s $(pwd)/www /var/www/rovercode/www
+rm -f /etc/nginx/sites-enabled/rovercode > /dev/null
+ln -s $(pwd)/nginx-site /etc/nginx/sites-enabled/rovercode
```

Comment:
You could add the `-f` option to `ln` to overwrite the link if it's already there and then remove the `rm` above